src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1208,26 +1216,50 @@ protected function applyScope($scope, $builder)
     }
 
     /**
-     * Determine if the scope added after the given offset should be nested.
+     * Alias for the shouldNestWheres method.
      *
      * @param  \Illuminate\Database\Query\Builder  $query
      * @param  int  $originalWhereCount
      * @return bool
      */
     protected function shouldNestWheresForScope(QueryBuilder $query, $originalWhereCount)
     {
+        return $this->shouldNestWheres($query, $originalWhereCount);
+    }
+
+    /**
+     * Determine if the scope added after the given offset should be nested.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @param  int  $originalWhereCount
+     * @return bool
+     */
+    protected function shouldNestWheres(QueryBuilder $query, $originalWhereCount)
+    {
```

Comment:
@wleona3 I don't think we really need to alias ``shouldNestWheresForScope``. Just use the existing method.